lib/electrode-server.js

```diff
@@ -24,8 +25,8 @@ function showRegisterPluginsNote(timeout) {
   const next = Chalk.red("next()");
   return setTimeout(() =>
       /* eslint-disable */
```

Comment:
please remove the `eslint-disable` marks

Reply:
OK.